src/styles/components/_header.scss

```diff
@@ -0,0 +1,69 @@
+.header {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  box-shadow: rgba(33, 35, 38, 0.1) 0 10px 10px -10px;
+  &__logo-link {
+    display: flex;
+    align-items: center;
+  }
+
+  &__nav-list {
+    display: flex;
+    list-style: none;
+    justify-content: center;
+    margin: 0;
+    padding: 0;
```

Comment:
`reset.scss` (margin, padding and list-style)

Reply:
Added    margin: 0; padding: 0; to normalize.scss and removed from all the other files